parser/block_header.go

```diff
@@ -11,7 +11,7 @@ import (
 	"math/big"
 	"unsafe"
 
-	verushash "github.com/Asherda/Go-VerusHash"
+	verushash "github.com/DavidLDawes/Go-VerusHash"
 	"github.com/Asherda/lightwalletd/parser/internal/bytestring"
```

Comment:
See comments on transition to Asherda fork then VerusCoin fork above. That will happen in later PRs.